app/app.js

```diff
@@ -8,7 +8,7 @@ require('angular-resource');
 require('angular-translate');
 require('angular-translate-loader-static-files');
 require('angular-ui-bootstrap');
-require('angular-datepicker');
+require('angular-datepicker/build/angular-datepicker');
```

Comment:
Try this as just `require(‘angular-datepicker’)` - then it loads whatever is set as `main` in package.json for that package